Deprecate `*syntax` options from `declaration-property-value-no-unknown` (#9102)
stylelint/stylelint: lib/rules/declaration-property-value-no-unknown/README.md

### `propertiesSyntax`

> [!WARNING]
> We've **deprecated** this option and will remove it in the next major release. Use the [`languageOptions`](../../../docs/user-guide/configure.md#languageoptions) configuration property instead.
>
> For example:
>
> ```diff json

> > +{

Author: Done

### `typesSyntax`

> [!WARNING]

> > This option is **deprecated** and we will remove it in the next major release. Use the [`languageOptions`](../../../docs/user-guide/configure.md#languageoptions) configuration property instead.

Reviewer: ```suggestion
> We've **deprecated** this option and will remove it in the next major release. Use the [`languageOptions`](../../../docs/user-guide/configure.md#languageoptions) configuration property instead.
```

Let's use a consistent active voice. (Same for other option)
